web/beacon-app/src/features/topics/components/TopicQueryResult/ViewingEvent.tsx

```diff
@@ -3,28 +3,28 @@ import { Heading } from '@rotational/beacon-core';
 
 import { CardListItem } from '@/components/common/CardListItem';
 
+import { getEventsPaginationCounter, getQueryPaginationCounter } from '../../utils';
 import MetaDataTable from './MetaDataTable';
-
 interface MetaDataProps {
   totalResults: number;
   totalEvents: string;
+  counter: number;
 }
 
-const ViewingEvent = ({ totalResults, totalEvents }: MetaDataProps) => {
+const ViewingEvent = ({ totalResults, totalEvents, counter }: MetaDataProps) => {
   return (
     <div className="mt-8">
       <Heading as="h2" className="mb-2 text-lg font-semibold">
         <Trans>
-          Query Results
-          <span className="ml-1 font-normal">
-            {' '}
-            - {String(totalResults ?? 'N/A')} of {totalEvents ?? 'N/A'}
+          Query Results -
+          <span className="ml-1 font-normal" data-testid="query-result-count">
```

Comment:
The number of results listed next to `Query Results` should be the total number of results returned from the query search and then the `total_events` number returned from the BE.  These numbers should not change when the user clicks `Previous` or `Next`.

`total_events` could be more than 10, but the BE will only return a maximum of 10 results for display to the user. 
 Providing the `total_events` in this way will help a user know if they have more than 10 results and should then help them understand that they will need to use the SDK if they want to view the other events not displayed in the results.